pkg/k8s/apis/ciliumendpoints.cilium.io/register.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpointciliumio
+
+const (
+	// GroupName is the group name of the CiliumEndpoint
+	GroupName = "cilium.io"
```

Comment:
This is `ciliumendpoints.cilium.io`

Reply:
Also, so we will go with `ciliumendpoints.cilium.io`? for the network policy I was thinking on `cnp.cilium.io` but, for the `CustomResourceDefinitionSingularName` I used `ciliumnetworkpolicy`

Reply:
The user is never going to type these things so I suggest to use long names that are clear. Abbreviations  always require to educate the user first.

Reply:
I'll try again but I couldn't figure out why the https://github.com/cilium/cilium/pull/3164 PR can use `cnp.cilium.io` without the resulting CRD being `ciliumnetworkpolicies.cnp.cilium.io`. I was aiming to keep the same name we have now. If I made `GroupName = "cilliumendpoints.cilium.io"` the CRD could not be registered because it wants the group field in the CRD to be `CustomResourceDefinitionPluralName + "." + SchemeGroupVersion.Group` which resulted in `ciliumendpoints.ciliumendpoint.cilium.io`. I don't fully understand why this works as it is (or doesn't, for that matter).
If we did intend to namespace all things endpoint related under ciliumendpoint.cilium.io then it might be better to go with the abbreviation for that middle bit.